src/components/BookList.js

```diff
@@ -12,9 +13,14 @@ const AvailablilityIcon = ({checkedOutBy}) => {
 }
 
 const BookLink = (props) => {
-    const { bookId, title, checkedOutBy } = props.book;
+    const { bookId, title, checkedOutBy, isbn } = props.book;
+    const bookData = useBookData(isbn);
+
+    const imageUrl = (bookData && bookData.cover) ? bookData.cover.small : undefined;
```

Comment:
javascript has added a null safe dereference operator. I wonder If our version of react-scripts supports it?

`bookData?.cover?.small`

Reply:
Unfortunately, it doesn't look like it. We could add it through a babel pre-compile step

Reply:
Probably not worth it then.  React-scripts 3.3 is supposed to add it